internal/controller/scheduler/scheduler.go

```diff
@@ -461,8 +461,7 @@ func (w *worker) Build(podSpec *corev1.PodSpec, skipCheckout bool, inputs buildI
 			c.EnvFrom = append(c.EnvFrom, inputs.k8sPlugin.GitEnvFrom...)
 
 			// Add annotations to the Job indicating which of the user-defined
```

Comment:
Noting that this code to add new annotations was added in #591, but they're not likely to be a problem.